### PR TITLE
Sound Duration Addon

### PR DESCRIPTION
**Pull request by TheColaber**
This addon is the most requested addon in the community server https://discord.com/channels/806602307750985799/806609030935740477/930480070768873482

### Changes

Really simple addon that traps methods created by the scratch-gui. It creates a display that shows how long the whole sound/selected part is and how long has been played:
image
image

I want to improve the styling and position so if you have recommendations, lemme know.

### Tests

Tested on chrome.

### CODE REVIEW

File: addons/sounds-duration/addon.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Sound Duration",
```
Comment (apple502j) [inline addons/sounds-duration/addon.json:2]:
```suggestion
  "name": "Sound duration display",
```

File: addons/sounds-duration/addon.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "description": "Displays the duration of sounds in the sound editor as well as how long it has been playing.",
+  "credits": [
+    {
+      "name": "TheColaber",
+      "link": "http://scratch.mit.edu/users/TheColaber"
```
Comment (apple502j) [inline addons/sounds-duration/addon.json:7]:
```suggestion
      "link": "https://scratch.mit.edu/users/TheColaber/"
```

**Review by apple502j — CHANGES_REQUESTED** [review]

File: addons/sounds-duration/addon.json (hunk 1 above)
Comment (TheColaber) [reply addons/sounds-duration/addon.json:2]: I don't think we use "display" in any addon names... We don't call it clone counter display

Comment (Secret-chest) [reply addons/sounds-duration/addon.json:2]: @TheColaber a counter implies a display

**Comment by CST1229** [comment]
Maybe show the selection duration above the selection area and the play time below the play button?

**Comment by RIMOPA** [comment]
I think this is a bug, when I select a part of the sound, after playing It the addon displays the duration of the whole sound instead of the duration of the selected part:

https://user-images.githubusercontent.com/97112832/155629103-c0a93a73-dccd-4711-a8d1-12782cdae81d.mp4

**Comment by WorldLanguages** [comment]
@TheColaber Is styling the only blocker for this PR?

**Comment by TheColaber** [comment]
Yep 👍

**Comment by WorldLanguages** [comment]
@mxmou could you help a bit with the styling?

**Comment by mxmou** [comment]
@WorldLanguages I changed the styling slightly:
image

Clicking on the waveform without dragging to make a selection changes the displayed duration to 0 until the sound is played again. That probably isn't intended.

The description should probably mention that it also shows the duration of the selection.

**Comment by WorldLanguages** [comment]
> Clicking on the waveform without dragging to make a selection changes the displayed duration to 0 until the sound is played again. That probably isn't intended.

This can be fixed after this PR is merged, as another PR, while we give translators time

> The description should probably mention that it also shows the duration of the selection.

Great point, I'll make changes to the manifest.

Merging tomorrow.

**Pull request merged**